cmd/stellar-rpc/internal/integrationtest/get_ledger_entries_test.go

```diff
@@ -120,7 +132,7 @@ func TestGetLedgerEntriesSucceeds(t *testing.T) {
 	require.Equal(t, xdr.LedgerEntryTypeContractCode, firstEntry.Type)
 	require.Equal(t, infrastructure.GetHelloWorldContract(), firstEntry.MustContractCode().Code)
 
-	require.Positive(t, result.Entries[1].LastModifiedLedger)
```

Comment:
the linter is suggesting to use `require.Positive()` instead of `require.Greater()`

Reply:
Yup, and if I use require.Positive, it says to use require.Greater.

Reply:
Example is here: https://stellarfoundation.slack.com/archives/C02B04RMK/p1747313010371909?thread_ts=1747312845.342179&cid=C02B04RMK